extensions/localPublish/src/index.ts

```diff
@@ -36,12 +38,14 @@ interface PublishConfig {
 
 const isWin = process.platform === 'win32';
 
-class LocalPublisher {
+class LocalPublisher implements PublishPlugin<PublishConfig> {
+  public name = 'localPublisher';
+  public description = 'Publish bot to local runtime';
   static runningBots: { [key: string]: RunningBot } = {};
```

Comment:
nit: Record<string, RunningBot>

Reply:
I always forget about `Record`. However, since this PR is scoped to the publishing name and description, I am going to leave as is.